crypto/dilithium/pqcrystals_dilithium_ref_common/packing.c

```diff
@@ -208,7 +208,7 @@ void pack_sig(ml_dsa_params *params,
 
   k = 0;
   for(i = 0; i < params->k; ++i) {
-    for(j = 0; j < N; ++j) {
+    for(j = 0; j < ML_DSA_N; ++j) {
```

Comment:
Missed updating the header file guard in packing.h

Reply:
[2bbdfc9](https://github.com/aws/aws-lc/pull/2072/commits/2bbdfc975539e79985135f515caa45399229d1f6)